src/constants/avatar.ts

```diff
@@ -99,3 +101,32 @@ export type AvatarEyebrowType = (typeof AVATAR_EYEBROW_TYPES)[number];
 
 export const AVATAR_MOUTH_TYPES = ["Concerned", "Default", "Disbelief", "Eating", "Grimace", "Sad", "ScreamOpen", "Serious", "Smile", "Tongue", "Twinkle", "Vomit"] as const;
 export type AvatarMouthType = (typeof AVATAR_MOUTH_TYPES)[number];
+
+export const AVATAR_CONFIG: AvatarConfig = {
+  hair: [
+    {values: AVATAR_TOP_TYPES, key: "topType"},
+    {
+      values: AVATAR_HAIR_COLORS,
+      key: "hairColor",
+      disabledOn: {topType: ["NoHair", "Eyepatch", "Hat", "Hijab", "Turban", "WinterHat1", "WinterHat2", "WinterHat3", "WinterHat4", "LongHairFrida", "LongHairShavedSides"]},
+    },
+    {values: AVATAR_FACIAL_HAIR_TYPES, key: "facialHairType", disabledOn: {topType: ["Hijab"]}},
+    {values: AVATAR_FACIAL_HAIR_COLORS, key: "facialHairColor", disabledOn: {topType: ["Hijab"], facialHairType: ["Blank"]}},
+  ],
+  facialFeatures: [
+    {values: AVATAR_SKIN_COLORS, key: "skinColor"},
+    {values: AVATAR_EYEBROW_TYPES, key: "eyebrowType"},
+    {values: AVATAR_EYE_TYPES, key: "eyeType"},
+    {values: AVATAR_MOUTH_TYPES, key: "mouthType"},
+  ],
+  clothing: [
+    {values: AVATAR_ACCESSORIES_TYPES, key: "accessoriesType", disabledOn: {topType: ["Eyepatch"]}},
+    {values: AVATAR_CLOTHE_TYPES, key: "clotheType"},
```

Comment:
If this PR is intended as a general refactoring – maybe rename the constant to `AVATAR_CLOTHING_TYPES` / `AVATAR_CLOTHES_TYPES`– because everything else is also a noun and not a verb.

Reply:
Although you're technically correct, the property names were taken from the underlying library. While changing the property keys is no problem is the frontend, the backend would be tricky as the json is stored like that in the backend, which in turn would require some sort of migration to change all existing instances of keys in the json `clotheColor` to `clothingColor`. 

So for a small refactor like this which was just intended to make TS happy I'm actually not to keen on implementing new logic, what do you think?